types/formidable/index.d.ts

```diff
@@ -29,6 +32,20 @@ export declare class IncomingForm extends events.EventEmitter {
     parse(req: http.IncomingMessage, callback?: (err: any, fields: Fields, files: Files) => any): void;
 }
 
+export declare interface IncomingFormOptions {
```

Comment:
This seems to be missing couple of props, reading the defaults, `enabledPlugins`,:
https://github.com/node-formidable/formidable/blob/82671664d1f333948dc96a2f8da38f7863973e88/src/Formidable.js#L17-L30
https://github.com/node-formidable/formidable/blob/82671664d1f333948dc96a2f8da38f7863973e88/src/Formidable.js#L45
https://github.com/node-formidable/formidable/blob/82671664d1f333948dc96a2f8da38f7863973e88/README.md#L527

Reply:
there is `filename` function options within the constructor implementation, not listed in defaults:
```js
const hasRename = typeof this.options.filename === 'function';
```
https://github.com/node-formidable/formidable/blob/82671664d1f333948dc96a2f8da38f7863973e88/src/Formidable.js#L64
so, it needs to be in the options IMO